src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -32,6 +32,8 @@ protocol StoreProtocol: AnyObject {
 	/// his diagnosiskeys to the CWA submission service.
 	var exposureActivationConsentAccept: Bool { get set }
 
+	var exposureDetectionDate: Date? { get set }
```

Comment:
1. Please add a comment for what/ why this is needed
2. A better naming would be good (and would resolve 1.). Perhaps `softRateExposureDetectionDate`?